sdk/compute/Microsoft.Azure.Management.Compute/tests/ScenarioTests/SharedGalleryTests.cs

```diff
@@ -0,0 +1,185 @@
+﻿using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using Microsoft.Azure.Management.ResourceManager.Models;
+using Microsoft.Azure.Management.ResourceManager;
+using System.Diagnostics;
+using Microsoft.Azure.Management.Compute.Models;
+using Microsoft.Rest.Azure;
+using System.Linq;
+using Microsoft.Azure.Management.Compute;
+
+
+namespace Compute.Tests
+{
+    public class SharedGalleryTests : VMTestBase
+    {
+        // these needs to be decided and created, especially subscription.
+        // two gallery shared to the tenant, but only one gallery shared to the sub
+        protected const string GalleryUniqueName = "97f78232-382b-46a7-8a72-964d692c4f3f-LONGLIVEGALLERYFOJNVV";
+        protected const string GalleryImageName = "jmaesscc";
+        protected const string GalleryImageVersionName = "1.0.0";
+        
+        private string galleryAccessLocation = "eastus2euap";
```

Comment:
Please use a non-Canary region for these tests.

Reply:
same as above